docker/Dockerfile.k8s

```diff
@@ -38,14 +32,6 @@ RUN set -ex; \
     percona-release setup psmdb-50; \
```

Comment:
And what about upgrading to `psmdb-60`?
Given that:
1. PMB may have to interact with PSMDB v6 instances
2. `percona-server-mongodb-shell` v6 is backwards compatible with older versions (I suppose)